src/ffi.rs

```diff
@@ -21,10 +20,10 @@ impl NewtonPoint {
 
```

Comment:
what does the line 22? #[no_mangle] do ?

Reply:
`#[no_mangle]` is an annotation that tells the rust compiler not to change the name of this function when it compiles it. Usually a compiler will change the function names so it doesn't conflict with other function names. But because we want to call this function externally, we need to use the name as it is defined